- name: "Tables"
  remove_surface: true
  parent_frame_id: "base_link"
  child_frame_id: "head_camera_depth_frame"
  bounding_frame_id: "base_link"
  segmentation_frame_id: "base_link"
  z_min: 0.10
  x_min: 0.0
  x_max: 3.0
